runelite-client/src/main/java/net/runelite/client/plugins/loginscreen/LoginScreenPlugin.java

```diff
@@ -308,4 +343,36 @@ private SpritePixels getFileSpritePixels(String file)
 
 		return null;
 	}
+
+	/**
+	 * Takes an image file and returns a SpritePixels object of the image data
+	 *
+	 * @param file A File object for the image
+	 * @return A SpritePixels object containing the image data
+	 */
+	private SpritePixels getFileSpritePixels(File file)
+	{
+		try
+		{
+			BufferedImage image;
+			synchronized (ImageIO.class)
+			{
+				image = ImageIO.read(file);
+			}
+
+			if (image.getHeight() > Constants.GAME_FIXED_HEIGHT)
+			{
+				final double scalar = Constants.GAME_FIXED_HEIGHT / (double) image.getHeight();
+				image = ImageUtil.resizeImage(image, (int) (image.getWidth() * scalar), Constants.GAME_FIXED_HEIGHT);
+			}
+
+			return ImageUtil.getImageSpritePixels(image, client);
+		}
+		catch (IOException | NullPointerException e)
+		{
+			log.error("error loading custom login screen: " + file, e);
+			restoreLoginScreen();
```

Comment:
This method shouldn't be doing login screen operations.

Reply:
No I'm not sure why I added that there.

Reply:
Fixed on latest commit.